### ISSUE TITLE
Performance on speaker identification is good ,but on speaker verification is poor

### ISSUE THREAD

**Issue by zrtlemontree** [mravanelli/SincNet#68]
I'm using your model to train on TIMIT, my parameters are the same as yours ,such as lr、batchsize. I use the script named  ‘speaker_id.py ’ to train model, and ‘compute_d_vector.py’ to get embeddings of utterances. In my experiment, I use the cosine similarity to get score,and the performance on speaker identification is good (like your result opening in github ),but on speaker verification I cannot reach my expectations(so far my EER is 3%).So I want to know your EER on TIMIT(verification). Many thanks.

**Comment by mravanelli** [comment]
Sorry for my late reply. Which is the EER that you able to achive?

**Comment by zrtlemontree** [comment]
Thanks for your reply.
My EER is about 2.5%. I use the "speaker_id.py" script and AM-Softmax to train sincnet, use the "compute_d_vector.py" script to extract embedding, and then use the cosine similarity score to calculate the EER. I tried a variety of trials, such as a trial with a correct ratio of 9:1 , and 1:1, but the EER is about 2.5% or more.
I can basically reproduce the results of your "speaker_id.py" script.

Thank you.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Mirco Ravanelli"<notifications@github.com&gt;;
发送时间:&nbsp;2019年11月16日(星期六) 晚上10:59
收件人:&nbsp;"mravanelli/SincNet"<SincNet@noreply.github.com&gt;;
抄送:&nbsp;"张瑞腾"<570836446@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [mravanelli/SincNet] Performance on speaker identification is good ,but on speaker verification is poor (#68)




Sorry for my late reply. Which is the EER that you able to achive?

**Comment by Tommy-Ngx** [comment]
@zrtlemontree Can you please share and help me with verification task? after get d-vector, i am confused to do that? please help! if possible, could you show the code you did for the verification task? thank you very much! my email is tommyfederation@gmail.com . Cheers

**Comment by Shrijoy-Chowdhury** [comment]
Hi I have trained the model using speaker-=_id.py script.Next I want to test my model on new dataset, can anyone help me out with it. I am not able to understand how to do it
